airbase/pom.xml

```diff
@@ -339,6 +339,8 @@
                                     <!-- use the newer Jetty version of the servlet API -->
                                     <exclude>javax.servlet:javax.servlet-api</exclude>
                                     <exclude>org.eclipse.jetty.orbit:javax.servlet</exclude>
+                                    <exclude>org.eclipse.jetty.toolchain:jetty-servlet-api</exclude>
```

Comment:
We shouldn't exclude this one, as it's a separate artifact and Java package which might be needed by dependencies. It doesn't conflict with the new ones.